Src/Test.UnitTests.Sarif.PatternMatcher/TestLogger.cs

```diff
@@ -37,6 +38,7 @@ public void LogConfigurationNotification(Notification notification)
 
         public void LogToolNotification(Notification notification)
         {
+            ToolNotification = notification;
```

Comment:
>ToolNotification [](http://example.com/codeflow?start=12&length=16)

Please make this a list and follow the log pattern for results to initialize and add the notification.